packages/gestalt/src/Card.js

```diff
@@ -44,8 +48,7 @@ export default function Card({ active, children, image, onMouseEnter, onMouseLea
   };
 
   const classes = classnames(styles.card, {
-    // JS equality rules == null checks for `null` or `undefined` and leaves out `false`.
-    [styles.hover]: active || (active == null && hovered),
+    [styles.hover]: isNil(active) ? hovered : active,
```

Comment:
Boolean logic can be confusing, especially when boolean comparators are used with boolean prop values. This simplifies things: if `active` has a value,  use that. Otherwise we'll use the hover state.